docs/user/getting_started.rst

```diff
@@ -29,8 +29,10 @@ freshly installed Debian Bullseye system the following packages are required:
 
 * `git` (to get Gluon and other dependencies)
 * `python3`
+* `python3-dev`
+* `python3-pyelftools`
+* `python3-setuptools`
```

Comment:
Good point, i've opened #3424 because adding those there as well really should be it's own PR. :)